xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -198,6 +198,12 @@ def _skip_target(
         transitive_infos = transitive_infos,
     )
 
+    # Some keys are not applicable in schemes, we will filter them out here
+    _IGNORED_BAZEL_ENV_VARIABLES = [
+        "XCODE_VERSION_OVERRIDE",
+        "XCODE_VERSION",
+    ]
+
```

Comment:
I'm probably not coming across correctly here, sorry. This should be a global constant in the file, not defined in any function, and referenced instead of passed. Also, a dictionary with `None` values for quick lookup.